CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
 
 # Build Options
+option(UR_BUILD_EXAMPLES "Build example applications." ON)
```

Comment:
you can add info about this into Readme, [here](https://github.com/oneapi-src/unified-runtime?tab=readme-ov-file#cmake-standard-options)

Reply:
Done.